Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
What I was thinking of was making it possible to go back to data that has been read, as long as it hasn't already been overwritten by new data (which would take 0,5 s on 2 MiB/s). It would require us to do some additional bookkeeping, though, so that it isn't possible to go back to data from before the read started.